package.json

```diff
@@ -7,14 +7,15 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node --watch src/server.js",
-    "dev": "nodemon -e hbs,js -V src/server.js",
+    "dev": "nodemon -e hbs,js -i cypress/ -V src/server.js",
```

Comment:
This will prevent the application server from restarting when changes are made to cypress files.